src/backends/github/API.js

```diff
@@ -335,14 +359,38 @@ export default class API {
     });
   }
 
-  mergePR(headSha, number) {
-    return this.request(`${ this.repoURL }/pulls/${ number }/merge`, {
+  mergePR(pullrequest, objects) {
+    const headSha = pullrequest.head;
+    const prNumber = pullrequest.number;
+    console.log("%c Merging PR", "line-height: 30px;text-align: center;font-weight: bold"); // eslint-disable-line
+    return this.request(`${ this.repoURL }/pulls/${ prNumber }/merge`, {
       method: "PUT",
       body: JSON.stringify({
         commit_message: "Automatically generated. Merged on Netlify CMS.",
         sha: headSha,
       }),
+    })
+    .catch((error) => {
+      if (error instanceof APIError && error.status === 405) {
+        this.forceMergePR(pullrequest, objects);
+      } else {
+        throw error;
+      }
+    });
+  }
+
+  forceMergePR(pullrequest, objects) {
+    const files = objects.files.concat(objects.entry);
+    const fileTree = this.composeFileTree(files);
+    let commitMessage = "Automatically generated. Merged on Netlify CMS\n\nForce merge of:";
+    files.forEach((file) => {
+      commitMessage += `\n* "${ file.path }"`;
     });
+    console.log("%c Automatic merge not possible - Forcing merge.", "line-height: 30px;text-align: center;font-weight: bold"); // eslint-disable-line
```

Comment:
I would log them with `console.error` so we know they're supposed to be there if we just grep for `console.log`